packages/contracts-bridge/contracts/test/TokenRegistry.t.sol

```diff
@@ -18,7 +18,7 @@ contract TokenRegistryTest is BridgeTestFixture {
     using TypedMemView for bytes29;
     using BridgeMessage for bytes29;
 
-    function setUp() public override {
+    function setUp() public virtual override {
```

Comment:
if it just invokes super, this should actually be deleted

Reply:
Makes sense